### PR TITLE
Update manage-tags.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6048
Added to note.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-tags.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ As mentioned earlier, there are two kinds of tags in Teams.
...
 
 > [!NOTE]
-> Tags are not yet supported in private channels. Tags are not yet available in US Government Community Cloud (GCC), GCC High, or Department of Defense (DoD) organizations.
+> Tags are not yet supported in private channels. Tags are not yet available in US Government Community Cloud (GCC), GCC High, or Department of Defense (DoD) organizations. Managing tags via Powershell is not yet available.
```
Comment (JohanFreelancer9) [inline Teams/manage-tags.md:36]:
```suggestion
> Tags are not yet supported in private channels. Tags are not yet available in US Government Community Cloud (GCC), GCC High, or Department of Defense (DoD) organizations. Managing tags via PowerShell is not yet available.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by LanaChin** [comment]
Moved note to the "Manage" section. Reviewed and OKed by PM.

**Pull request merged**